components/Form/index.jsx

```diff
@@ -29,48 +60,97 @@ function Form({ onSubmit, expense = [], isEditMode, setToast }) {
       <StyledTitle>{isEditMode ? "Edit Expense" : "Add Expense"}</StyledTitle>
       <form onSubmit={onSubmit}>
         <StyledMargin>
+          <StyledLabel htmlFor="name">Expense Title</StyledLabel>
           <StyledInput
+            $isValid={isWrong.name ? false : true}
             id="name"
             name="name"
             type="text"
-            placeholder="Title"
-            maxLength={100}
+            placeholder="Supermarket"
+            maxLength={25}
+            minLength={3}
             required
             defaultValue={isEditMode ? expense.name : ""}
+            onBlur={(event) => {
+              formValidation(event)
+                ? setIsWrong({ name: false })
+                : setIsWrong({ name: true });
+            }}
```

Comment:
Here we can do a couple of things. 

1. We can call the `formValidation` function and store the result in a variable. This way we don't need to conditionally call `setIsWrong`. 
2. The second this we can do is spread the old elements object into the new one when we call `setIsWrong`. That way more than one input field can we invalid at once. This may or may not be something you want to do. 

```suggestion
            onBlur={(event) => {
              const isValid = formValidation(event);
              setIsWrong((elements) => ({ ...elements, name: !isValid }));
            }}
```